### DISCUSSION TITLE
WebFlasher appears to not progress to Stage 2

### DISCUSSION THREAD

**Discussion by HeWhoWantsJeansAgain** [eh2k/squares-and-circles#95] [General]
I'm doing a pass through my studio to update hardware firmware and decided to update my Teensy 4.0 o_C builds. They were previously running Square and Circles, so I we have a known-good starting point. When I go through the WebFlasher, I get to what I think should be Stage 2 but it does not proceed farther. The "Flashing Squares&Circlesl Loader" seems to have progressed fine but it reboots the unit and the browser just says: "Ready: Squares&Circles ()" and the front display on the o_C flashes "Ready to Update" but nothing happens.

Has anyone else experienced this behavior? I'd like to sort this successfully on one of my o_C units before trying the second.

Thank you,
Erik

**Comment by jpnielsen** [comment]
There's an issue thread going on

https://github.com/eh2k/squares-and-circles/issues/66

**Reply by HeWhoWantsJeansAgain** [reply]
Thanks! I'll poke over there.

**Comment by eh2k** [comment]
Closing this issue.

The WebMidi interface is used in stage 2. Please close other applications that could use midi in the background to be on the safe side.
